core/db/entities.py

```diff
@@ -31,309 +34,258 @@ def to_dict(self):
 
-    __tablename__ = "procurement"
+    __tablename__ = "funding_question"
 
-    construction_contract = sqla.Column(sqla.String(), nullable=False)
-    project_id: Mapped[int] = sqla.orm.mapped_column(sqla.ForeignKey("project_dim.id"), nullable=False)
-    start_date = sqla.Column(sqla.DateTime(), nullable=False)
-    end_date = sqla.Column(sqla.DateTime(), nullable=False)
-    status = sqla.Column(sqla.Enum(const.StatusEnum, name="status"), nullable=False)
-    procurement_status = sqla.Column(sqla.Enum(const.ProcurementStatusEnum, name="procurement_status"), nullable=False)
+    submission_id: Mapped[int] = sqla.orm.mapped_column(sqla.ForeignKey("submission_dim.id"), nullable=False)
+    programme_id: Mapped[int] = sqla.orm.mapped_column(sqla.ForeignKey("programme_dim.id"), nullable=False)
 
-    # TODO: Should this be nullable? Is Status alone enough in some circumstances?
-    comments = sqla.Column(sqla.String(), nullable=False)
+    question = sqla.Column(sqla.String(), nullable=False)
+    indicator = sqla.Column(sqla.String(), nullable=False)
+    response = sqla.Column(sqla.String(), nullable=True)
+    guidance_notes = sqla.Column(sqla.String(), nullable=True)
 
-    # TODO: does this unique index look right?
-    # Unique index for data integrity.
     __table_args__ = (
         sqla.Index(
-            "ix_unique_procurement",
-            "construction_contract",
-            "project_id",
+            "ix_unique_funding_question",
+            "submission_id",
+            "programme_id",
+            "question",
+            "indicator",
             unique=True,
         ),
     )
 
 
-class ProjectProgress(BaseModel):
-    """Stores Project Progress answers."""
+class Project(BaseModel):
+    """Stores Project Entities."""
 
-    __tablename__ = "project_progress"
+    __tablename__ = "project_dim"
 
-    package_id: Mapped[int] = sqla.orm.mapped_column(sqla.ForeignKey("package_dim.id"), nullable=False)
+    project_id = sqla.Column(sqla.String(), nullable=False, unique=False)
+    submission_id: Mapped[int] = sqla.orm.mapped_column(sqla.ForeignKey("submission_dim.id"), nullable=False)
+    programme_id: Mapped[int] = sqla.orm.mapped_column(sqla.ForeignKey("programme_dim.id"), nullable=True)
 
-    answer_1 = sqla.Column(sqla.String(), nullable=True)
-    answer_2 = sqla.Column(sqla.String(), nullable=True)
-    answer_3 = sqla.Column(sqla.String(), nullable=True)
-    answer_4 = sqla.Column(sqla.String(), nullable=True)
-    answer_5 = sqla.Column(sqla.String(), nullable=True)
-    answer_6 = sqla.Column(sqla.String(), nullable=True)
+    project_name = sqla.Column(sqla.String(), nullable=False)
+    primary_intervention_theme = sqla.Column(sqla.String(), nullable=False)
+    location_multiplicity = sqla.Column(
+        sqla.Enum(const.MultiplicityEnum, name="project_location_multiplicity"), nullable=False
+    )
+    # locations = sqla.Column(sqla.ARRAY(sqla.String), nullable=False)  # ARRAY not supported by SQLite
```

Comment:
thinking about it, let's not use the array at all - that reply's far to much on users entering in some valid format. Just a string is fine (can remove the comment).